models/prices/prices_stability_rank.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(
+        schema='prices',
+        alias = 'stablibity_rank',
```

Comment:
instead of `stability_rank`, i think we simplify to just `trusted_tokens` or `core_tokens`